src/shipping/src/shipping_service/quote.rs

```diff
@@ -25,19 +21,23 @@ pub async fn create_quote_from_count(count: u32) -> Result<Quote, tonic::Status>
         }
     };
 
+    let meter = global::meter("items_count_meter");
```

Comment:
for a follow up PR - create meter, counter once and re-use instead of repeatedly creating: https://github.com/open-telemetry/opentelemetry-rust/blob/main/docs/metrics.md#metrics-api